# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# Dušan Kazik <prescott66@gmail.com>, 2020
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-09-28 22:49+0200\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: Dušan Kazik <prescott66@gmail.com>, 2020\n"
"Language-Team: Slovak (https://app.transifex.com/calamares/teams/20061/sk/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: sk\n"
"Plural-Forms: nplurals=4; plural=(n % 1 == 0 && n == 1 ? 0 : n % 1 == 0 && n >= 2 && n <= 4 ? 1 : n % 1 != 0 ? 2: 3);\n"

#: src/modules/bootloader/main.py:46
msgid "Install bootloader."
msgstr "Inštalácia zavádzača."

#: src/modules/bootloader/main.py:644
msgid "Failed to install grub, no partitions defined in global storage"
msgstr ""

#: src/modules/bootloader/main.py:899
msgid "Bootloader installation error"
msgstr ""

#: src/modules/bootloader/main.py:900
msgid ""
"The bootloader could not be installed. The installation command "
"<pre>{!s}</pre> returned error code {!s}."
msgstr ""

#: src/modules/displaymanager/main.py:509
msgid "Cannot write LXDM configuration file"
msgstr "Nedá sa zapísať konfiguračný súbor správcu LXDM"

#: src/modules/displaymanager/main.py:510
msgid "LXDM config file {!s} does not exist"
msgstr "Konfiguračný súbor správcu LXDM {!s} neexistuje"

#: src/modules/displaymanager/main.py:598
msgid "Cannot write LightDM configuration file"
msgstr "Nedá sa zapísať konfiguračný súbor správcu LightDM"

#: src/modules/displaymanager/main.py:599
msgid "LightDM config file {!s} does not exist"
msgstr "Konfiguračný súbor správcu LightDM {!s} neexistuje"

#: src/modules/displaymanager/main.py:684
msgid "Cannot configure LightDM"
msgstr "Nedá s nakonfigurovať správca LightDM"

#: src/modules/displaymanager/main.py:685
msgid "No LightDM greeter installed."
msgstr "Nie je nainštalovaný žiadny vítací nástroj LightDM."

#: src/modules/displaymanager/main.py:716
msgid "Cannot write SLIM configuration file"
msgstr "Nedá sa zapísať konfiguračný súbor správcu SLIM"

#: src/modules/displaymanager/main.py:717
msgid "SLIM config file {!s} does not exist"
msgstr "Konfiguračný súbor správcu SLIM {!s} neexistuje"

#: src/modules/displaymanager/main.py:935
msgid "No display managers selected for the displaymanager module."
msgstr "Neboli vybraní žiadni správcovia zobrazenia pre modul displaymanager."

#: src/modules/displaymanager/main.py:936
msgid ""
"The displaymanagers list is empty or undefined in both globalstorage and "
"displaymanager.conf."
msgstr ""

#: src/modules/displaymanager/main.py:1023
msgid "Display manager configuration was incomplete"
msgstr "Konfigurácia správcu zobrazenia nebola úplná"

#: src/modules/dracut/main.py:29
msgid "Creating initramfs with dracut."
msgstr "Vytváranie initramfs pomocou nástroja dracut."

#: src/modules/dracut/main.py:63
msgid "Failed to run dracut"
msgstr ""

#: src/modules/dracut/main.py:64
#, python-brace-format
msgid "Dracut failed to run on the target with return code: {return_code}"
msgstr ""

#: src/modules/dummypython/main.py:35
msgid "Dummy python job."
msgstr "Fiktívna úloha jazyka python."

#: src/modules/dummypython/main.py:37 src/modules/dummypython/main.py:93
#: src/modules/dummypython/main.py:94
msgid "Dummy python step {}"
msgstr "Fiktívny krok {} jazyka python"

#: src/modules/fstab/main.py:28
msgid "Writing fstab."
msgstr "Zapisovanie fstab."

#: src/modules/fstab/main.py:377 src/modules/fstab/main.py:383
#: src/modules/fstab/main.py:411 src/modules/initcpiocfg/main.py:256
#: src/modules/initcpiocfg/main.py:260 src/modules/initramfscfg/main.py:85
#: src/modules/initramfscfg/main.py:89 src/modules/localecfg/main.py:140
#: src/modules/mount/main.py:329 src/modules/networkcfg/main.py:105
#: src/modules/openrcdmcryptcfg/main.py:72
#: src/modules/openrcdmcryptcfg/main.py:76 src/modules/rawfs/main.py:164
msgid "Configuration Error"
msgstr "Chyba konfigurácie"

#: src/modules/fstab/main.py:378 src/modules/initramfscfg/main.py:86
#: src/modules/mount/main.py:330 src/modules/openrcdmcryptcfg/main.py:73
#: src/modules/rawfs/main.py:165
msgid "No partitions are defined for <pre>{!s}</pre> to use."
msgstr "Nie sú určené žiadne oddiely na použitie pre <pre>{!s}</pre>."

#: src/modules/fstab/main.py:384 src/modules/initramfscfg/main.py:90
#: src/modules/localecfg/main.py:141 src/modules/networkcfg/main.py:106
#: src/modules/openrcdmcryptcfg/main.py:77
msgid "No root mount point is given for <pre>{!s}</pre> to use."
msgstr "Nie je zadaný žiadny bod pripojenia na použitie pre <pre>{!s}</pre>."

#: src/modules/fstab/main.py:412
msgid "No <pre>{!s}</pre> configuration is given for <pre>{!s}</pre> to use."
msgstr ""

#: src/modules/grubcfg/main.py:29
msgid "Configure GRUB."
msgstr "Konfigurácia zavádzača GRUB."

#: src/modules/hwclock/main.py:26
msgid "Setting hardware clock."
msgstr "Nastavovanie hardvérových hodín."

#: src/modules/initcpiocfg/main.py:27
msgid "Configuring mkinitcpio."
msgstr "Konfigurácia mkinitcpio."

#: src/modules/initcpiocfg/main.py:257
msgid "No partitions are defined for <pre>initcpiocfg</pre>."
msgstr ""

#: src/modules/initcpiocfg/main.py:261
msgid "No root mount point for <pre>initcpiocfg</pre>."
msgstr ""

#: src/modules/initramfscfg/main.py:32
msgid "Configuring initramfs."
msgstr "Konfigurácia initramfs."

#: src/modules/localecfg/main.py:31
msgid "Configuring locales."
msgstr "Konfigurácia miestnych nastavení."

#: src/modules/mkinitfs/main.py:27
msgid "Creating initramfs with mkinitfs."
msgstr ""

#: src/modules/mkinitfs/main.py:49
msgid "Failed to run mkinitfs on the target"
msgstr ""

#: src/modules/mkinitfs/main.py:50
msgid "The exit code was {}"
msgstr "Kód skončenia bol {}"

#: src/modules/mount/main.py:43
msgid "Mounting partitions."
msgstr "Pripájanie oddielov."

#: src/modules/mount/main.py:164 src/modules/mount/main.py:200
msgid "Internal error mounting zfs datasets"
msgstr ""

#: src/modules/mount/main.py:176
msgid "Failed to import zpool"
msgstr ""

#: src/modules/mount/main.py:192
msgid "Failed to unlock zpool"
msgstr ""

#: src/modules/mount/main.py:209 src/modules/mount/main.py:214
msgid "Failed to set zfs mountpoint"
msgstr ""

#: src/modules/mount/main.py:365
msgid "zfs mounting error"
msgstr ""

#: src/modules/networkcfg/main.py:29
msgid "Saving network configuration."
msgstr "Ukladanie sieťovej konfigurácie."

#: src/modules/openrcdmcryptcfg/main.py:26
msgid "Configuring OpenRC dmcrypt service."
msgstr ""

#: src/modules/packages/main.py:54 src/modules/packages/main.py:65
#: src/modules/packages/main.py:75
msgid "Install packages."
msgstr "Inštalácia balíkov."

#: src/modules/packages/main.py:63
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "Spracovávajú sa balíky (%(count)d / %(total)d)"

#: src/modules/packages/main.py:68
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] "Inštaluje sa jeden balík."
msgstr[1] "Inštalujú sa %(num)d balíky."
msgstr[2] "Inštaluje sa %(num)d balíkov."
msgstr[3] "Inštaluje sa %(num)d balíkov."

#: src/modules/packages/main.py:71
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] "Odstraňuje sa jeden balík."
msgstr[1] "Odstraňujú sa %(num)d balíky."
msgstr[2] "Odstraňuje sa %(num)d balíkov."
msgstr[3] "Odstraňuje sa %(num)d balíkov."

#: src/modules/packages/main.py:740 src/modules/packages/main.py:752
#: src/modules/packages/main.py:780
msgid "Package Manager error"
msgstr ""

#: src/modules/packages/main.py:741
msgid ""
"The package manager could not prepare updates. The command <pre>{!s}</pre> "
"returned error code {!s}."
msgstr ""

#: src/modules/packages/main.py:753
msgid ""
"The package manager could not update the system. The command <pre>{!s}</pre>"
" returned error code {!s}."
msgstr ""

#: src/modules/packages/main.py:781
msgid ""
"The package manager could not make changes to the installed system. The "
"command <pre>{!s}</pre> returned error code {!s}."
msgstr ""

#: src/modules/plymouthcfg/main.py:27
msgid "Configure Plymouth theme"
msgstr "Konfigurácia motívu služby Plymouth"

#: src/modules/rawfs/main.py:26
msgid "Installing data."
msgstr "Inštalácia údajov."

#: src/modules/services-openrc/main.py:29
msgid "Configure OpenRC services"
msgstr "Konfigurácia služieb OpenRC"

#: src/modules/services-openrc/main.py:57
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr ""

#: src/modules/services-openrc/main.py:59
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr ""

#: src/modules/services-openrc/main.py:61
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""

#: src/modules/services-openrc/main.py:93
msgid "Cannot modify service"
msgstr "Nedá sa upraviť služba"

#: src/modules/services-openrc/main.py:94
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""

#: src/modules/services-openrc/main.py:101
msgid "Target runlevel does not exist"
msgstr ""

#: src/modules/services-openrc/main.py:102
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""

#: src/modules/services-openrc/main.py:110
msgid "Target service does not exist"
msgstr "Cieľová služba neexistuje"

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr "Cesta k službe {name!s} je <code>{path!s}</code>, ale neexistuje."

#: src/modules/services-systemd/main.py:26
msgid "Configure systemd units"
msgstr ""

#: src/modules/services-systemd/main.py:64
msgid "Cannot modify unit"
msgstr ""

#: src/modules/services-systemd/main.py:65
msgid ""
"<code>systemctl {_action!s}</code> call in chroot returned error code "
"{_exit_code!s}."
msgstr ""

#: src/modules/services-systemd/main.py:66
msgid "Cannot {_action!s} systemd unit <code>{_name!s}</code>."
msgstr ""

#: src/modules/unpackfs/main.py:34
msgid "Filling up filesystems."
msgstr "Napĺňanie súborových systémov."

#: src/modules/unpackfs/main.py:254
msgid "rsync failed with error code {}."
msgstr "Príkaz rsync zlyhal s chybovým kódom {}."

#: src/modules/unpackfs/main.py:299
msgid "Unpacking image {}/{}, file {}/{}"
msgstr "Rozbaľuje sa obraz {}/{}, súbor {}/{}"

#: src/modules/unpackfs/main.py:314
msgid "Starting to unpack {}"
msgstr "Spúšťa sa rozbaľovanie {}"

#: src/modules/unpackfs/main.py:323 src/modules/unpackfs/main.py:467
msgid "Failed to unpack image \"{}\""
msgstr "Zlyhalo rozbalenie obrazu „{}“"

#: src/modules/unpackfs/main.py:430
msgid "No mount point for root partition"
msgstr "Žiadny bod pripojenia pre koreňový oddiel"

#: src/modules/unpackfs/main.py:431
msgid "globalstorage does not contain a \"rootMountPoint\" key."
msgstr ""

#: src/modules/unpackfs/main.py:434
msgid "Bad mount point for root partition"
msgstr "Zlý bod pripojenia pre koreňový oddiel"

#: src/modules/unpackfs/main.py:435
msgid "rootMountPoint is \"{}\", which does not exist."
msgstr ""

#: src/modules/unpackfs/main.py:439 src/modules/unpackfs/main.py:455
#: src/modules/unpackfs/main.py:459 src/modules/unpackfs/main.py:465
#: src/modules/unpackfs/main.py:480
msgid "Bad unpackfs configuration"
msgstr ""

#: src/modules/unpackfs/main.py:440
msgid "There is no configuration information."
msgstr ""

#: src/modules/unpackfs/main.py:456
msgid "The filesystem for \"{}\" ({}) is not supported by your current kernel"
msgstr "Súborový systém pre \"{}\" ({}) nie je podporovaný vaším aktuálnym jadrom"

#: src/modules/unpackfs/main.py:460
msgid "The source filesystem \"{}\" does not exist"
msgstr "Zdrojový súborový systém \"{}\" neexistuje"

#: src/modules/unpackfs/main.py:466
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed."
msgstr ""

#: src/modules/unpackfs/main.py:481
msgid "The destination \"{}\" in the target system is not a directory"
msgstr "Cieľ \"{}\" v cieľovom systéme nie je adresárom"

#: src/modules/zfshostid/main.py:27
msgid "Copying zfs generated hostid."
msgstr ""
